agent/lm_agent/config.py

```diff
@@ -11,6 +11,7 @@
 
 
 DEFAULT_DOTENV_PATH = Path("/etc/default/license-manager-agent")
+DEFAULT_CACHE_DIR = Path.home() / Path(".cache/license-manager")
```

Comment:
```suggestion
DEFAULT_CACHE_DIR = Path("/var/cache/license-manager")
```
?

Reply:
No, it needs to default to home for running locally. In the charm, it is set to `/var/cache/license-manager`

Reply:
I don't quite understand. The `DEFAULT_CACHE_DIR` is just the default setting. The final value will either be the same or can be overriden by setting the environment.